src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,95 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix6.configs.CANcoderConfiguration;
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.signals.AbsoluteSensorRangeValue;
+import com.ctre.phoenix6.signals.SensorDirectionValue;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import static frc.trigon.robot.utilities.Conversions.offsetRead;
+
+public class ArmConstants {
+    private static final int
+            MASTER_ANGLE_MOTOR_ID = 1,
+            MASTER_ELEVATOR_MOTOR_ID = 2,
+            FOLLOWER_ANGLE_MOTOR_ID = 3,
+            FOLLOWER_ELEVATOR_MOTOR_ID = 4,
+            ANGLE_ENCODER_ID = 5,
+            ELEVATOR_ENCODER_ID = 6;
+    private static final CANSparkMax.IdleMode
+            ELEVATOR_IDLE_MODE = CANSparkMax.IdleMode.kBrake,
+            ANGLE_IDLE_MODE = CANSparkMax.IdleMode.kBrake;
+    private static final int VOLTAGE_COMPANSATION_SATURATION = 12;
+    private static final boolean INVERTED_VALUE = false;
```

Comment:
The problem I had with this previously was not only the name, but the fact you created only one for 4 motors. 
No need for the _VALUE part since it's not an inverted value